helpers/z_schema.js

```diff
@@ -208,7 +208,7 @@ var liskFormats = {
 	 * @todo Add description for the function, the params and the return value
 	 */
 	queryList(obj) {
-		if (obj == null || typeof obj !== 'object' || _.isArray(obj)) {
+		if (!obj || typeof obj !== 'object' || _.isArray(obj)) {
```

Comment:
Same as explained above, changing obj == null to !obj is not correct.

Here the condition is checking if a value is not an object or an array, we can do the same by:
`if(!_.isObject(obj) || _.isArray(obj))` or even simpler by `if (_.isArray(obj))`. Explanation:
```
typeof null
"object"
typeof undefined
"undefined"
typeof {}
"object"
typeof [] 
"object"

_.isObject(null)
false
_.isObject(undefined)
false
_.isObject({})
true

_.isArray([])
true
_.isArray(null)
false
_.isArray(undefined)
false
_.isArray({})
false
``

Reply:
@MaciejBaj `typeof obj !== 'object' || Array.isArray(obj)` I'd prefer to use native solutions instead of depending on lodash. What do you think?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
I couldn't agree more @yatki